articles/active-directory/develop/quickstart-v2-java-webapp.md

```diff
@@ -150,10 +136,9 @@ This quickstart application uses a client secret to identify itself as confident
 
 ### Getting MSAL
 
-MSAL4J is the library used to sign in users and request tokens used to access an API protected by the Microsoft identity
-Platform.
-You can add MSAL4J to your application by using Maven or Gradle to manage your dependencies by making the following changes to
-the pom.xml or build.gradle file in your application.
+MSAL4J is the library used to sign in users and request tokens used to access an API protected by the Microsoft identity Platform.
```

Comment:
I think you need to clarify that it is the Java library for doing this.

Reply:
ok edited